### PR TITLE
Fix smart contract interface

### PR DESCRIPTION
**Pull request by kbeker**
Small fixes which we discussed with @cameel

### CODE REVIEW

File: concent_api/concent_api/settings/base.py [hunk 1]
```diff
@@ -233,10 +233,18 @@
 STORAGE_CLUSTER_SSL_CERTIFICATE_PATH = ''
 
 # A global constant defining amount of time needed for geth to create new chain segment
-CREATION_NEW_CHAIN_SEGMENT_TIME = 15  # seconds
+# AVERAGE_BLOCK_TIME = 15  # seconds
 
 # A global constant defining address to geth client
-GETH_CONTAINER_ADDRESS = 'http://localhost:8545'
+# GETH_CONTAINER_ADDRESS = 'http://localhost:8545'
```
Comment (cameel) [inline concent_api/concent_api/settings/base.py]: You can add `http://localhost:8545` in `development.py.

File: concent_api/concent_api/settings/base.py [hunk 2]
```diff
@@ -233,10 +233,18 @@
...
+
+# A global constant defining Concent ethereum address
+# Stored in a 'string' 0x...
+# CONCENT_ETHEREUM_ADDRESS = ''
```
Comment (cameel) [inline concent_api/concent_api/settings/base.py]: Please clarify that it's the address of Concent's contract.

File: concent_api/concent_api/system_check.py [hunk 3]
```diff
@@ -156,19 +156,19 @@ def storage_cluster_certificate_path_check(app_configs = None, **kwargs):  # pyl
 
 
 @register()
-def creation_new_chain_segment_time_check(app_configs, **kwargs):  # pylint: disable=unused-argument
+def creation_average_block_time_check(app_configs, **kwargs):  # pylint: disable=unused-argument
```
Comment (cameel) [inline concent_api/concent_api/system_check.py]: Why not `average_block_time_check`?

File: concent_api/core/payments/sci_backend.py [hunk 4]
```diff
@@ -20,24 +22,24 @@ def get_list_of_payments(
     assert isinstance(provider_eth_address,     str) and len(provider_eth_address)  == ETHEREUM_ADDRESS_LENGTH
     assert isinstance(payment_ts,               int) and payment_ts     >= 0
     assert isinstance(current_time,             int) and current_time   > 0
-    assert isinstance(transaction_type,         str) and transaction_type in ['Force', 'Batch']
+    assert isinstance(transaction_type,         Enum) and transaction_type.name in ['Force', 'Batch']
```
Comment (cameel) [inline concent_api/core/payments/sci_backend.py]: This should be `transaction_type in TransactionType`

File: concent_api/concent_api/settings/base.py (hunk 1 above)
Comment (kbeker) [reply concent_api/concent_api/settings/base.py]: Done

File: concent_api/concent_api/settings/base.py (hunk 2 above)
Comment (kbeker) [reply concent_api/concent_api/settings/base.py]: Done

File: concent_api/concent_api/system_check.py (hunk 3 above)
Comment (kbeker) [reply concent_api/concent_api/system_check.py]: Done

File: concent_api/core/payments/sci_backend.py (hunk 4 above)
Comment (kbeker) [reply concent_api/core/payments/sci_backend.py]: Done

**Pull request merged**